pkg/skaffold/runner/dev.go

```diff
@@ -151,10 +151,5 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 		}
 	}
 
-	// Forward ports
-	if err := forwarderManager.Start(ctx); err != nil {
```

Comment:
would this be effective change as moving this at L147?  and add at L70 again. its easier to read since, 
- you stopped portforwading 
- did a deploy
- start port forwarding again.

The deploy does not get cluttered with portforward management.
WDYT?

Reply:
that makes sense, thanks! just changed it.